launch/record_rosbag.launch

```diff
@@ -0,0 +1,5 @@
+<launch>
+
+    <node pkg="agimus_controller" name="record_rosbag" type="record_rosbag" />
```

Comment:
no don't do that like this.
https://answers.ros.org/question/52773/record-with-rosbag-from-launch-file/
I would go for:
```xml
<launch>
 <node pkg="rosbag" type="record" name="rosbag_record_mpc"
       args="record -o $(env ROS_LOG_DIR)/rosbag_record_mpc /ctrl_mpc_linearized/ocp_solve_time"
 />
</launch>
```